fake-server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require("body-parser");

```

Comment:
Used double quotes here but single on the line above

Reply:
thanks! I'll standardise on double quotes for this code